### PR TITLE
Enable adding tau.circle.css for circle projects

### PR DESCRIPTION
**Pull request by korneliakobiela**
Problem: Checkbox wasn't shown in Circle Wearable way.
Solution: Add tau.circle.css library when it doesn't exist and the project should be a circle.

Signed-off-by: Kornelia Kobiela <k.kobiela@samsung.com>

### CODE REVIEW

File: design-editor/src/pane/design-editor.js [hunk 1]
```diff
@@ -944,7 +944,10 @@ class Model {
 
 		this._DOM.body.appendChild(scriptRunner);
 		this.addLibrary('helper.js');
-		this.addLibrary('circle-helper.js');
+		if (screen.shape == 'circle') {
```
Comment (hsiwkin) [inline design-editor/src/pane/design-editor.js]: why `==` and not `===`?

File: design-editor/src/system/libraries/js-library.js [hunk 2]
```diff
@@ -32,44 +31,16 @@ class JSLibrary extends Library {
...
-	setAttribute(key, value='') {
-		this.addAttribute(key,value);
-		this.element.setAttribute(key, value);
-	}
-
-	createDataAttribute() {
-		return `data-${basename(this._fileName, extname(this._fileName))}`;
-	}
-
 	getSelector() {
```
Comment (hsiwkin) [inline design-editor/src/system/libraries/js-library.js]: can we move getSelector to the Library class?

File: design-editor/src/system/libraries/library.js [hunk 3]
```diff
@@ -90,6 +91,15 @@ class Library {
 		});
 	}
 
+	/**
+	 * Set element attribute
+	 * @param {string} key attribute key
+	 * @param {string} value attribute value
+	 */
+	setAttribute(key, value='') {
+		this.addAttribute(key,value);
```
Comment (hsiwkin) [inline design-editor/src/system/libraries/library.js]: please add space after `,`

File: design-editor/src/system/libraries/js-library.js (hunk 2 above)
Comment (korneliakobiela) [reply design-editor/src/system/libraries/js-library.js]: For now, I leave it as is. It's a good point to refactor in the next iteration

**Pull request merged**